main-app/app/scripts/game/services/bingo-ticket.js

```diff
@@ -1,41 +1,52 @@
-(function () {
+(function() {
     'use strict';
     angular.module('Tombola.Games.Bingo90.Core')
         .service('BingoTicket', [function() {
-            var me = this;
-            var tickets = [];
+            var me = this,
+                tickets = [],
+                lines = [];
 
             me.balls = [];
+            me.markedNumbers = [];
 
-            me.pushArray = function (card) {
-                var cardss = "054963758028345266770611596982054963758028345266770611596982";
-                generateTheTicket(card);
-            };
-
-            var generateTheTicket = function (ticket) {
-                var startOfString = 0;
-                var endOfString = 30;
-                for(startOfString; startOfString < ticket.length; startOfString += 30){
-                    tickets.push(ticket.slice(startOfString, endOfString));
-                    endOfString += 30;
-                }
-
-                for(ticket in tickets){
-                    splitUpTheTicket(tickets[ticket]);
+            me.pushArray = function(card) {
+                var cardss = "054963758028345266770611596982054963758028345266770611596982",
```

Comment:
Really? Hardcoded? 
Also `card` not `cards`
Also this is almost useless from a programmatic point of view - a whacking great string in place of what is really a 10x4 grid.